### DISCUSSION TITLE
Permissions issue with new V9 APIs

### DISCUSSION THREAD

**Discussion by anand-kulk** [directus/directus#3493] [Archive]
Hi there,
   I am testing 9.0.0-rc-26 and using postgres as a backend. I can get auth token with auth/login call.
But for any further attempt to get collection or any items under it, I get below error, I have a collection with name first. I am calling API as admin user. Also find attached postman screenshot. The issue was there on rc19 too, thinking this is specific to postgres.
Appreciate any help to resolve this.

```
{
    "errors": [
        {
            "message": "You don't have permission to access the \"first\" collection.",
            "extensions": {
                "code": "FORBIDDEN"
        }
        }
    ]
}
```
![permission issue with APIs](https://user-images.githubusercontent.com/70729249/102859396-ac74c080-43e0-11eb-9579-92f682f014d0.png)

**Comment by rijkvanzanten** [comment]
What does Postman do with that "Bearer Token" type? Do you have to manually add `Bearer ` again in that input? 

Could you be sending an invalid auth header?

**Reply by anand-kulk** [reply]
@rijkvanzanten, It adds Authorization request header with "Bearer <token>" value. Yes, I manually added that field from what was received as access_token in auth response.

**Reply by anand-kulk** [reply]
@rijkvanzanten , I tried with Curl and issue is still there. Attaching the token I got from auth and my curl calls below - 

![access token response](https://user-images.githubusercontent.com/70729249/102915244-53d01280-4436-11eb-8d45-ec70630d6f5d.png)

![Issues with Curl](https://user-images.githubusercontent.com/70729249/102915270-5af72080-4436-11eb-9f31-a5adb695ed85.png)

**Reply by anand-kulk** [reply]
@rijkvanzanten Looks like the issue is resolved for me. I was entering Bearer in postman for Bearer Auth and it was inserting duplicate "Bearer" in authorization header. I removed it and now I get response correctly. Thanks for pointing it out! I've had no issues with that extra word with some other APIs though.
Attaching successful requests made for others who had similar issues - 
![Get all items in collection](https://user-images.githubusercontent.com/70729249/102917985-fee2cb00-443a-11eb-92ad-cbd2a3ce15ad.png)
![Get item in collection](https://user-images.githubusercontent.com/70729249/102917995-05714280-443b-11eb-8606-1160a75a13e2.png)
